experiments/AMIP/modular/Project.toml

```diff
@@ -52,16 +52,15 @@ ArgParse = "1.1"
 ArtifactWrappers = "0.2"
 AtmosphericProfilesLibrary = "0.1"
 CLIMAParameters = "0.4, 0.5, 0.6, 0.7"
-ClimaAtmos = "0.9"
+ClimaAtmos = "0.10"
 ClimaComms = "0.3"
 ClimaCommsMPI = "0.3"
 ClimaCore = "0.10"
 ClimaCorePlots = "0.2"
 ClimaCoreTempestRemap = "0.3"
 ClimaCoupler = "0.1"
-ClimaLSM = "0.2"
```

Comment:
Why did you remove the ClimaLSM compat?

Reply:
It slipped. Thanks for the catch. I put it back